lib/src/visitor/async_evaluate.dart

```diff
@@ -512,7 +539,62 @@ final class _EvaluateVisitor
             configuration: configuration,
             namesInErrors: true);
         _assertConfigurationIsEmpty(configuration, nameInError: true);
-      }, url: "sass:meta")
+      }, url: "sass:meta"),
+      BuiltInCallable.mixin("apply", r"$mixin, $args...", (arguments) async {
+        var mixin = arguments[0];
+        var args = arguments[1] as SassArgumentList;
+
+        var callableNode = _callableNode!;
+        var invocation = ArgumentInvocation(
+          [],
+          {},
+          callableNode.span,
+          rest: ValueExpression(args, callableNode.span),
+        );
+
+        var callable = mixin.assertMixin("mixin").callable;
+        var content = _environment.content;
+
+        switch (callable) {
+          case AsyncBuiltInCallable(acceptsContent: false) when content != null:
+            throw _exception(
+                "Mixin doesn't accept a content block.", callableNode.span);
```

Comment:
We should be able to use the same `MultiSpanSassRuntimeException` as the user-defined case, although you may need to make `BuiltInCallback.overloads` public to do so.